source/develop/release-management/features/virt/coreos-ignition-support-design-discussion

```diff
@@ -0,0 +1,69 @@
+---
+title: CoreOS Ignition Support Design Review
+category: feature
+authors: srosenbe
+feature_name: CoreOS Ignition Support Design Review
+feature_modules: engine
+feature_status: Tech Preview 4.3
+---
+
+## Overview
+Design Proposals based upon CoreOS Ignition Support Documentation: 
+
+https://github.com/oVirt/ovirt-site/edit/master/source/develop/release-management/features/virt/coreos-ignition-support.md
+
+The purpose of this document is to define which features shall be supported by CoreOS Ignition for RHCOS and FCOS within oVirt as per the current Cloud-Init implementation. As per the document referenced, certain features may not be included at least within the initial release and therefore an agreement on what may be added initially and what can be implemented in later releases is required as per the "Limitations" and "Future work" sections therein.
```

Comment:
It should be noted that the 'support' scope here is UI and API. It needs to be clearly understood that we support 
passing the ignition config v2 as is already,  and the added features here meant to generate the ignition config themselves, rather than just passing it to custom_script.